ui/pages/users/index.js

```diff
@@ -119,45 +119,43 @@ export default function Users () {
       <Head>
         <title>Users - Infra</title>
       </Head>
-      {loading
-        ? <Loader />
-        : (
-          <div className='flex-1 flex h-full'>
-            <main className='flex-1 flex flex-col space-y-4'>
-              <PageHeader header='Users' buttonHref={admin && '/users/add'} buttonLabel='User' />
-              {error?.status
-                ? <div className='my-20 text-center font-light text-gray-300 text-sm'>{error?.info?.message}</div>
-                : (
-                  <div>
-                    <Table
-                      {...table}
-                      getRowProps={row => ({
-                        onClick: () => setSelectedUser(row.original),
-                        style: {
-                          cursor: 'pointer'
-                        }
-                      })}
-                    />
-                    {users?.length === 0 && <EmptyTable
-                      title='There are no users'
-                      subtitle='Invite users to Infra and manage their access.'
-                      iconPath='/users.svg'
-                      buttonHref={admin && '/users/add'}
-                      buttonText='Users'
-                                            />}
-                  </div>
-                  )}
-            </main>
-            {selectedUser &&
-              <Sidebar
-                handleClose={() => setSelectedUser(null)}
-                title={selectedUser.name}
-                profileIcon={selectedUser.name[0]}
-              >
-                <SidebarContent selectedUser={selectedUser} admin={admin} setSelectedUser={setSelectedUser} />
-              </Sidebar>}
-          </div>
-          )}
+      {!loading && (
```

Comment:
we can remove the import Loader from this file then since we are not using it anymore

Reply:
great catch!